api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -175,7 +189,7 @@ export async function internalGetDocuments({
     if (d.content.size === 0) {
       log(`Document is of size 0, this may result in a 404 error - doc id ${d.id}`);
       capture.message("Document is of size 0", {
-        extra: d.content,
+        extra: { document: d },
```

Comment:
That's a good point. Sentry will capture just the first 2 levels, just like `console.log("xxx", error)`, example:

<img width="213" alt="image" src="https://github.com/metriport/metriport/assets/2132564/bb6a28fb-0bbe-42a9-9883-add78245206f">

Which is not a bad thing by itself... the payload can be quite large, and we've been logging the full doc ref on CloudWatch... what do you think? I'm fine with this, but can update to use stringify if you feel strongly about it.

Reply:
Since we've had issues before where we couldn't debug sufficiently due to things not being stringified, and since the doc ref payload won't be that large, I'd opt for using stringify